Cython/Compiler/PyrexTypes.py

```diff
@@ -4464,6 +4464,9 @@ def __init__(self, cname, components):
         self.exception_check = True
         self._convert_to_py_code = None
         self._convert_from_py_code = None
+        # equivalent_type must be set now because it isn't available at import time
+        from . import Builtin
+        self.equivalent_type = Builtin.tuple_type
```

Comment:
If `tuple_type` is really all we need, then let's just import that directly.
```suggestion
        from .Builtin import tuple_type
        self.equivalent_type = tuple_type
```